slim-java.sh

```diff
@@ -14,17 +14,45 @@
 #
 set -o pipefail
 
-# We only support 64 bit builds now
-proc_type="64bit"
+# Parse arguments
+argc=$#
+if [ ${argc} != 1 ]; then
+	echo " Usage: `basename $0` Full-JDK-path"
+	exit 1
+fi
+
+# Validate prerequisites(tools) necessary for making a slim build
+tools="jar jarsigner pack200 strip"
```

Comment:
FYI in case you missed it...
http://openjdk.java.net/jeps/336

Reply:
This should only be run for Java 8 binaries